x-pack/plugins/security_solution/cypress/tasks/api_calls/elasticsearch.ts

```diff
@@ -16,6 +16,61 @@ export const deleteIndex = (index: string) => {
   });
 };
 
+export const createIndex = (indexName: string, properties: Record<string, unknown>) =>
```

Comment:
I see this same function here: `x-pack/plugins/security_solution/cypress/tasks/api_calls/risk_scores/indices.ts`. Could we update the other instances of it to just use this one or vice versa? Probably this one since it's in a more central folder.

Reply:
I also found that other implementation. But it looks like it is using a different API endpoint: not a public ES API, but an internal Kibana endpoint: `/internal/risk_score/indices`. I thought this endpoint might behave differently - do something specific to risk scoring. What do you think?

Reply:
Ah, good catch! Closing out.